docs/workforce-management-framework/topics/con-production-checklist.adoc

```diff
@@ -1,19 +1,19 @@
 [id='{context}-production-checklist']
 = Production checklist
 
-Following guide provides set of guidance and actions that needs to be verified before releasing RainCatcher to production.
+Following guide provides set of actions that needs to be executed before releasing RainCatcher to production.
 
 == Server configuration
 
-Server application provides abstraction for configuration that allows users to create profiles.
+Server application provides abstraction for configuration that allows users to create profiles for different environments
 Application is created with 2 default profiles:
 
 - development (link:{WFM-RC-ServerURL}{WFM-RC-Release-Tag}/config-dev.js[config-dev.js])
 - production (link:{WFM-RC-ServerURL}{WFM-RC-Release-Tag}/config-prod.js[config-prod.js])
 
 For more information about profiles go to link:{context}-ref-server[Server side reference]
 
-Whem moving to production developers should review their configuration to make sure that a
+Whem moving to production developers should review server configuration.
```

Comment:
Spelling ```When```
When moving to production developers should review server configuration.